### PR TITLE
Setup/c setup

### PR DESCRIPTION
**Pull request by Failte21**
Les fichiers qui t'intéressent:
- Makefile (pas celui de la libft)
- ft_p.h
- gitignore

Possible améliorations:
- Un Makefile qui recompile lorsque le .h change
- Une architecture 'domain': 'client/srcs' au lieu de 'srcs/clients'

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+$(CLIENT_OBJS_DIR)/%.o: $(SRCS_DIR)/$(CLIENT_DIR)/%.c
+	@mkdir -p $(CLIENT_OBJS_DIR)
+	$(CC) $(CFLAGS) -c -o $@ $^
+
+$(COMMON_OBJS_DIR)/%.o: $(SRCS_DIR)/$(COMMONS_DIR)/%.c
+	@mkdir -p $(COMMON_OBJS_DIR)
+	$(CC) $(CFLAGS) -c -o $@ $^
+
+clean:
+	@echo "make clean -C libft/"
```
Comment (ekelen) [inline Makefile]: Needs to remove `objs` folder created in root

File: README.md [hunk 2]
```diff
@@ -36,4 +36,28 @@ $ docker-compose exec pureftpd bash
 If the ftp client is not available on your version of MacOS, `brew install inetutils`
 
 #### Connect to FTP container
-$ ftp server_ip
\ No newline at end of file
+$ ftp server_ip
+
+#### C implementaion
```
Comment (ekelen) [inline README.md]: This should be a level two subheading (`##`) and come before my testing instructions

File: Makefile (hunk 1 above)
Comment (Failte21) [reply Makefile]: 6bc983bb66bdf96187503fbc2d0f072b8b589d84

File: README.md (hunk 2 above)
Comment (Failte21) [reply README.md]: a64b6da674678d5735e26015b1360a97b93fcebd

**Pull request merged**